install_bootstrap

```diff
@@ -285,6 +285,9 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
         aws_default_region='aws_default_region'
         ask_question "What is your AWS region?" "$AWS_DEFAULT_REGION" "AWS region" $aws_default_region
 
+        azul_s3_aws_region='azul_s3_aws_region'
+        ask_question "What is the AWS region where the BDBag for export to FireCloud will temporarily be stored?" "$AZUL_S3_AWS_REGION" "Azul_S3_Aws_Region" $azul_s3_aws_region
```

Comment:
I would leave `FireCloud` out of it, and it should be plural.

Thinking about it more, because this applies to the azul_s3_bucket, I think you should move this below line 327, where you ask for the name of azul bucket, and simply ask something like`What is the AWS region of the S3 <bucket name user entered in answering previous question> bucket?`. Or if that's too tricky, reference the previous question more generically.